File: comms/bulk-messaging/messages/fetch-message/fetch-message.php
<?php

namespace Example;

use Twilio\Comms\TwilioComms;

$client = new TwilioComms(
    accountId: 'TWILIO_ACCOUNT_SID',
    authToken: 'TWILIO_AUTH_TOKEN',
);
$client->messages->fetch(
    'comms_message_01h9krwprkeee8fzqspvwy6nq8',
);

File: comms/bulk-messaging/messages/fetch-message/fetch-message.py
from twilio import TwilioComms

client = TwilioComms(
    account_id="<username>",
    auth_token="<password>",
)

client.messages.fetch(
    message_id="comms_message_01h9krwprkeee8fzqspvwy6nq8",
)
